src/react-components/info-dialog.js

```diff
@@ -155,6 +156,35 @@ class InfoDialog extends Component {
           </div>
         );
         break;
+      case InfoDialog.dialogTypes.safari:
+        dialogTitle = "Open in Safari";
+        dialogBody = (
+          <div>
+            <div>
+              Hubs is not supported in your current browser on iOS. Copy and paste this link directly in Safari.
```

Comment:
Nitpick: I think I'd prefer: "Hubs does not support your current browser on iOS."